src/test/java/emissary/test/core/junit5/UnitTest.java

```diff
@@ -267,6 +271,7 @@ protected Document getAnswerDocumentFor(String resource) {
     @Nullable
     protected Document getAnswerDocumentValidated(String aname) {
         SchemaFactory schemaFactory = SchemaFactory.newInstance(XMLConstants.W3C_XML_SCHEMA_NS_URI);
+        String answerXsd = getAnswerXsd();
```

Comment:
The local variable `answerXsd` has the same name as the field. Suggest calling `getAnswerXsd()` multiple times in this method or calling the field `answerXsd`